docs/rfcs/accepted/0007-hardening-memory-operations/README.md

```diff
@@ -53,50 +54,47 @@ checks can rapidly grow into a methodology that is easy to replicate as new code
 focus shall be made in layers that are close to the entry point, but subtle weaknesses can cross 
 many layers and effort should be done to track and sanitize memory transactions all the way down. 
 
-Memory types depend on the system, the platform, there is no standard memory ranges that are 
-either secure or unsecure. It can't be solved without the help of an info from the system the code 
-is executed on. The top level idea here is to implement similar logic as the Linux kernel does 
-when copying memory back and forth with userland -- with functions, ala copy_from_user(), 
+Memory types depend on the system and the platform as there are no standard memory ranges that are 
+either secure or unsecure. It can't be solved without the help of an information from the system the
+code is executed on. The top level idea here is to implement similar logic as the Linux kernel does 
+when copying memory back and forth with userland -- with functions, a la copy_from_user(),
 copy_to_user().
 
-The recommendation is to define an interface that aims at providing the memory type to of a given 
-memory block.
-For instance 'get_memory_type(address, size)' could return an enum from a list like {Shared, 
-SecureDataPath, SecureStack, SecureHeap, Invalid} and so forth.
-This would be a porting interface that would be asked for integrator partners to implement. 
-Note that, this routine shall check that all of the bytes from 'address', over 'size' bytes are 
-all inside of a given memory type (in other words, crossing memory types shall return an invalid 
-result)
-
-Then depending on the intent of a given memory operation, routines could add allowed memory 
-transaction sanitization.
+The recommendation is to define an interface that provides the memory type to of a given 
+memory block. For instance 'get_memory_type(address, size)' could return an enum from a list like
+{Shared, SecureDataPath, SecureStack, SecureHeap, Invalid} and so forth. This would be a porting
+interface that integrator partners to implement. Note that, this routine shall
```

Comment:
The line is short. Please check line lengths and rebalance paragraph wording to ensure maximum line lengths are utilized where ever possible.